### PR TITLE
Mock out imports directly so sphinx-click works

### PR DESCRIPTION
**Pull request by nvictus**

### CODE REVIEW

File: docs/conf.py [hunk 1]
```diff
@@ -14,6 +14,37 @@
...
 sys.path.insert(0, os.path.abspath('..'))
 
+from unittest.mock import Mock
+MOCK_MODULES = [
+    'cooltools.io.fastsavetxt',
+    'cooltools.lib._numutils',
+    'cooler',
+    'cooler.core',
+    'cooler.tools',
+    'cython',
```
Comment (sergpolly) [inline docs/conf.py]:
```suggestion
    'cython',
    'cytoolz',
```

File: docs/requirements.txt [hunk 2]
```diff
@@ -1,14 +1,9 @@
 bioframe
-cooler
 click>=7.0
-cython
 ipython
 nbsphinx
 numpy
```
Comment (sergpolly) [inline docs/requirements.txt]:
```suggestion
numpy
multiprocess
```

Comment (nvictus) [reply docs/requirements.txt]: Not `numpy` because there's a np.array literal in `plotting` right now that needs numpy to work.

Comment (sergpolly) [reply docs/requirements.txt]: I don't know how to request `+multiprocess` alone - this was meant to be 
```
-numpy
+numpy
+multiprocess
```

Comment (sergpolly) [reply docs/requirements.txt]: there are just tooo many buttons on github ...

Comment (nvictus) [reply docs/requirements.txt]: Yeah, I got confused with mock and requirements... I meant to keep numpy too.

**Pull request merged**